app/models/article.rb

```diff
@@ -27,6 +27,9 @@ class Article < ApplicationRecord
 
   # The date that we began limiting the number of user mentions in an article.
   MAX_USER_MENTION_LIVE_AT = Time.utc(2021, 4, 7).freeze
+  # The commented variable below is the image markdown regex for images stored in the local public folder.
+  # IMG_MARKDOWN_REGEX = %r{!\[.*\]\(#{ApplicationConfig["APP_PROTOCOL"]}#{ApplicationConfig["APP_DOMAIN"]}\/uploads\/articles\/(.*)\)}.freeze
```

Comment:
I don't think we should keep this in the code, what is the purpose of this regexp if it stays commented in the code?

Reply:
I left that in to verify image deletion for the images stored in the public folder while running on localhost. Same for the commented code in the test. But you're right, they aren't needed, I'll remove it.